src/stories/components/FileUpload/styles.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// FileUpload
+
+.file-upload-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.file-upload {
+  p {
+    display: inline-block;
+    margin-right: $spacing-1;
+  }
+}
+
+.upload-icon {
+  p {
```

Comment:
Can't nest block elements inside of inline ones. You can add the icon to the button by using a pseudo-element or using the icon as a background image.

Reply:
Is this ok now? I removed the img tag from inside of the p tag and used the icon as a background image for the p tag